Pod/Classes/MentionComposer.swift

```diff
@@ -262,14 +264,26 @@ public class MentionComposer<T: UIView where T: ComposableAttributedTextContaini
 
         // Check for a mention query
         if let query = mentionQuery(fromString: text) {
+            requestingMentionUsers = true
+            refreshTableView()
```

Comment:
Well the userNameMatchesArray is no longer the only factor in whether the table is hidden. `requestingMenitonUsers` is taken into account as well.